### DISCUSSION TITLE
Getting a OSError that i dont understand.

### DISCUSSION THREAD

**Discussion by timpop12** [carson-katri/dream-textures#693] [Q&A]
So im getting an OSError,

OSError('Error no file named pytorch_model.bin, tf_model.h5 or flax_model.msgpack found in directory C: ( my stable diffusion 2 depth path).

i've got no clue as to how to solve this on my own, hopefully its not too serious of an issue.

Thanks in advance

**Comment by 32kda** [comment]
For me the reason was that HuggingFace is using some folder under C:/Users to download models, and free space on C: just ended while downloaded
